### PR TITLE
feat(sprint): update for july scisprint

### PR DESCRIPTION
**Pull request by rockleona**

### CODE REVIEW

File: content/pages/index.rst [hunk 1]
```diff
@@ -10,11 +10,11 @@ home
     <h2 class="text-2xl text-center mb-5">Upcoming Events</h2>
     <div class="flex flex-row flex-wrap text-center">
         <div class="md:w-full w-full flex-col space-y-1 items-center">
-            <h3 class="text-lg">sciwork seminar summer 2024</h3>
+            <h3 class="text-lg">scisprint Taipei July 2024</h3>
             <div class="flex-row space-x-5">
-                <a href="/seminar/2024/06-seminar">Introduction</a>
+                <a href="/sprint/2024/07-sprint">Introduction</a>
```
Comment (kuomc) [inline content/pages/index.rst:15]: The href should be "/sprint/2024/07-taipei".

File: content/pages/sprint/2024/07-taipei.rst [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+  </div>
+
+|
+
+Contact us
+----------
+
+* sciwork: https://sciwork.dev/
+* Discord: https://discord.gg/6MAkFrD
+* Email: `contact@sciwork.dev (subject: I want to lead a project in scisprint) <mailto:contact@sciwork.dev?subject=[sciwork]%20I%20want%20to%20lead%20a%20project%20in%20scisprint>`__
```
Comment (kuomc) [inline content/pages/sprint/2024/07-taipei.rst:156]: All items should start with lowercase.

File: content/pages/sprint/2024/08-taipei.rst [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+  </div>
+
+|
+
+Contact us
+----------
+
+* sciwork: https://sciwork.dev/
+* Discord: https://discord.gg/6MAkFrD
+* Email: `contact@sciwork.dev (subject: I want to lead a project in scisprint) <mailto:contact@sciwork.dev?subject=[sciwork]%20I%20want%20to%20lead%20a%20project%20in%20scisprint>`__
```
Comment (kuomc) [inline content/pages/sprint/2024/08-taipei.rst:167]: All items should start with lowercase.

**Review by kuomc — CHANGES_REQUESTED** [review]
Please check the introduction link in the home page.

**Comment by rockleona** [comment]
@kuomc just updated, please have a look again

**Pull request merged**